otherlibs/stdune/ansi_color.mli

```diff
@@ -1,5 +1,43 @@
 module Style : sig
-  type t
+  type t = private
```

Comment:
why is it made private?

Reply:
`private` is for enforcing internal invariants that might be violated if the user constructs the type themselves. Doesn't seem like there's any such invariants in this api.

Reply:
Actually, why don't you leave the type abstract in this PR? you're not using its representation anywhere i can tell.